src/components/experiences/feedback-experiences.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { DialogContent, DialogActions } from '@material-ui/core';
+import ProgressBottom from '@general/navs/progress-bottom'; 
+
+import PrimaryButton from '@general/buttons/primary-button';  
+
+export default function FeedbackExperiences(props) {    
+ 
+    function showNextButton() { 
+      if(props.payload.tags.length === 0) { 
+        return false;
+      } else { 
+        return true;
+      }
+    } 
+
+    function handleSelection(tag) { 
+      let arr = props.payload.tags;
+
+      if(arr.includes(tag)){ 
+        arr = arr.filter( (item) => (!(item === tag)) );
+        
+        props.handler({ 
+          edited: props.payload.edited,
+          positive: props.payload.positive, 
+          tags: arr
+        });
+      } else { 
+        arr.push(tag);
+        props.handler({ 
+          edited: props.payload.edited,
+          positive: props.payload.positive, 
+          tags: arr
+        });
+      }
+    }
+
+    function renderTags() {  
+      let tagContainer = [];
+      for(let i=0; i<6; i+=2){ 
```

Comment:
per convention we usually use lodash foreach for iterating. not blocking this PR. why loop by 2s?